### ISSUE TITLE
Unable to go get latest pdfcpu release v0.4.0 to build into golang application

### ISSUE THREAD

**Issue by sereeena** [pdfcpu/pdfcpu#567]
OS is Ubuntu 20.04.5 LTS (focal)
I have upgraded to go version 1.20.1

Was previously using pdfcpu v0.3.13 successfully with go get github.com/pdfcpu/pdfcpu and using the API functions, but I wanted to upgrade to the latest version to use the Resize command. 

When I attempt upgrade I get:
```
go get -u github.com/pdfcpu/pdfcpu
go: github.com/pdfcpu/pdfcpu@v0.4.0: verifying go.mod: github.com/pdfcpu/pdfcpu@v0.4.0/go.mod: reading https://sum.golang.org/lookup/github.com/pdfcpu/pdfcpu@v0.4.0: 404 Not Found
````

Is pdfcpu no longer supported as a library? I don't want to install and use it via command line, it's much better to use the API functions within my golang application, please continue to allow this?

thank you

**Comment by jerem17** [comment]
Same thing here...

**Comment by hhrutter** [comment]
Sorry.. I need to look into this.

**Comment by hhrutter** [comment]
This issue is still under investigation.

workaround for now: `brew install pdfcpu`

**Comment by jerem17** [comment]
Yes but I can't use the API functions directly as if I had installed it with go get.

**Comment by hhrutter** [comment]
👍 my bad, `brew` only installs the cli.

`go get`seems to be running into timeout issues because of the reposize...

Follow the install instrucions in the Readme.md, this seems to be working:
```
git clone https://github.com/pdfcpu/pdfcpu
cd pdfcpu/cmd/pdfcpu
go install
```

**Comment by jerem17** [comment]
Work fine with go get now, thx a lot

**Comment by hhrutter** [comment]
👍  You got lucky... 
Occasional timeouts during `go get` may happen due to current repo size.
For these cases the workaround above still applies until further notice.

**Comment by sereeena** [comment]
go get started working and I was able to get v0.4.0, but the MergeRaw function is not available in the API for this version? The Merge function used to take ReadSeeker inputs and now requires filenames to read from disk.

I think it is supposed to be, as you have written about it in the Release Notes that we should use MergeRaw, but this is not possible?

**Comment by hhrutter** [comment]
You need to get the latest commit.

**Comment by lpic10** [comment]
`go get github.com/pdfcpu/pdfcpu@768cf964b3f08cf950399e0eeffab2ba2fe75485` did for me, it also updates the go.mod file

**Comment by sereeena** [comment]
Thank you @lpic10 that's what I needed, except I am getting the 404 timeout again during go get. I guess I will wait or keep trying.

Note @hhrutter your workaround with git clone and go install is not useful for me as I am using go modules and therefore need go get...

**Comment by hhrutter** [comment]
Try this in order to get the latest & greatest:
```
go env -w GOPROXY=direct
go env -w GOSUMDB=off
go get github.com/pdfcpu/pdfcpu@master
```

**Comment by sereeena** [comment]
Thank you, that seems to be solving the timeout issue!

**Comment by Lacutee** [comment]
just want add, i try `go get github.com/pdfcpu/pdfcpu/pkg/api` and its perfectly fine since package size seems only 2mib, no issue with repo size. which i found at this go dev documentation https://pkg.go.dev/github.com/pdfcpu/pdfcpu/pkg/api

**Comment by tgulacsi** [comment]
Putting samples somewhere else would reduce the repo size about 270MiB.
See discussion: https://groups.google.com/g/golang-nuts/c/q8hweiIanBQ

**Comment by hhrutter** [comment]
I am aware of the issue..

The problem is not so much the size of the samples but the fact that the repo size got out of hand because of commits including newer versions of the samples.

I am still looking for a way to get a clean repo with all samples - they are an important part of pdfcpu and the involved learning experience for engaged devs.

Until then please stick with the workaround mentioned above.

**Comment by tgulacsi** [comment]
Any odds against git-lfs for that two directories?

**Comment by tgulacsi** [comment]
Corundum

**Comment by hhrutter** [comment]
too much overhead
